storm-core/test/jvm/org/apache/storm/ui/StormApiResourceTest.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.storm.ui;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+
+import org.glassfish.grizzly.http.server.HttpServer;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class StormApiResourceTest {
+
+    private HttpServer server;
+    private WebTarget target;
```

Comment:
Is this just a place holder for a test or are we actually going to get some tests?

Reply:
I will add a single test here